.travis.yml

```diff
@@ -71,6 +71,8 @@ matrix:
     - python: 3.6
       env:
        - NUMPY_EXPERIMENTAL_ARRAY_FUNCTION=1
+    - os: linux-ppc64le
+      python: 3.6
```

Comment:
Maybe add a dummy `env:` variable to indicate this is ppc, the [test output matrix](https://travis-ci.org/numpy/numpy/builds/477580933) does not have job names so it is difficult to tell what is different about this run. Or is there a different way to label each job in the matrix?

Reply:
You can mouse over the penguin icon to find a ppc label, but good point nonetheless -- I've added a dummy env variable